tools/mo/unit_tests/mo/utils/cli_parser_test.py

```diff
@@ -469,6 +469,22 @@ def test_get_shapes_several_inputs_several_shapes2(self):
         for i in placeholder_values_ref.keys():
             npt.assert_array_equal(placeholder_values_res[i], placeholder_values_ref[i])
 
+    def test_get_shapes_and_freezing_with_scalar(self):
+        # shapes and value for freezing specified using --input command line parameter
+        argv_input = "inp1,inp2->157"
+        result_shapes, _ = get_placeholder_shapes(argv_input, None)
```

Comment:
please check data type also

Reply:
At this stage if user didn't specified data type we cannot check it. If user did't specified data type we will derive it during 
https://github.com/openvinotoolkit/openvino/blob/master/tools/mo/openvino/tools/mo/front/freeze_placeholder_value.py#L48-L57